docs/modules/ROOT/examples/operator/bundle.yaml

```diff
@@ -98,6 +95,109 @@ status:
   conditions: []
   storedVersions: []
 ---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.1
+  creationTimestamp: null
+  name: managementcenters.hazelcast.com
+spec:
+  group: hazelcast.com
+  names:
+    kind: ManagementCenter
+    listKind: ManagementCenterList
+    plural: managementcenters
+    singular: managementcenter
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: ManagementCenter is the Schema for the managementcenters API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
```

Comment:
```suggestion
            description: 'Versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
```

Reply:
This description is created by the tool that we use to create the yaml file, I think it would be better to leave it as is because we cannot change it in the other repository.